### ISSUE TITLE
Accessibility issues after upgrading to 1.26

### ISSUE THREAD

**Issue by didim99** [go-gitea/gitea#37448]
### Description

After upgrading to Gitea 1.26.1 it seems to be some accessibility problems with new UI in default dark theme:
- In diff page file line counter has a poor contrast ratio, especially for red (deletions) text with CR 3.73 (for green is 6.53 that slightly better but not enough), in previous versions CR was 11.47 in combination with bold text.
  For 1.25.5:
  <img width="1404" height="149" alt="Image" src="https://github.com/user-attachments/assets/e3408f5b-1bf9-4378-b829-c4a88272f57f" />
  For 1.26.1:
  <img width="1404" height="151" alt="Image" src="https://github.com/user-attachments/assets/08f8bd7b-6b00-45e7-96dc-ad0c07a5807b" />
  Github: CR 5.16 with bold font:
  <img width="1378" height="214" alt="Image" src="https://github.com/user-attachments/assets/70e1c097-5f37-406a-9373-f239e786aa98" />
  **Proposed solution**: At least add a bold font to lines counter (enough for existing green color), or slightly change red color to increase contrast ratio to 5 or better.
- Same issue with new syntax highlighting color scheme: new color for function names (`--color-syntax-name`) has a CR 5.76 (AA) when previous was a 8.03 (AAA):
  For 1.25.5:
  <img width="759" height="124" alt="Image" src="https://github.com/user-attachments/assets/e3bb5e6b-b3a0-4888-b1ee-0168de0cd7af" />
  For 1.26.1:
  <img width="758" height="107" alt="Image" src="https://github.com/user-attachments/assets/a861fcac-2803-4254-908d-38ef3cb1c12e" />
  **Proposed solution**: revert a previous color (`#fabd2f`) for function names and types

P. S. I have a vision problems and things like poor contrast ratio is not "just a numbers" but significantly affects everyday work.

### Screenshots

Embedded in description

### Gitea Version

1.26.1

### Can you reproduce the bug on the Gitea demo site?

No

### Operating System

Ubuntu 20.04/24.04

### Browser Version

Firefox 149

**Comment by silverwind** [comment]
1. Yes, red text color needs to be slightly brighter on dark theme, ideally same perceived lightness as the green. also need to do same for the colorblind variants.
2. Agree, bold is better there.
3. Agree, we can restore the previous yellow shade on syntax, it looked better. Also should do a pass over the other new syntax colors over green/red diff bgs to ensure sufficient contrast.

Goal should be a minimum of 7:1 CR (WCAG AAA).
